pageserver/src/metrics.rs

```diff
@@ -2225,17 +2293,13 @@ impl Drop for RemoteTimelineClientMetrics {
 pub(crate) trait MeasureRemoteOp: Sized {
     fn measure_remote_op(
         self,
-        tenant_id: TenantId,
-        timeline_id: TimelineId,
         file_kind: RemoteOpFileKind,
         op: RemoteOpKind,
         metrics: Arc<RemoteTimelineClientMetrics>,
     ) -> MeasuredRemoteOp<Self> {
         let start = Instant::now();
         MeasuredRemoteOp {
             inner: self,
-            tenant_id,
-            timeline_id,
             file_kind,
             op,
             start,
```

Comment:
Did we simply drop the tenant and timeline id labels?

Ah, I guess we already did in the past `294b8a8fde50418090c8fcb2b5c1c47e8cc11557`